misc/dbt-materialize/dbt/adapters/materialize/impl.py

```diff
@@ -36,3 +39,34 @@ def _link_cached_relations(self, manifest):
         #
         # [0]: https://github.com/dbt-labs/dbt-core/blob/13b18654f03d92eab3f5a9113e526a2a844f145d/plugins/postgres/dbt/adapters/postgres/impl.py#L126-L133
         pass
+
+    def verify_database(self, database):
+        pass
+
+    def list_relations_without_caching(
+        self, schema_relation: MaterializeRelation
+    ) -> List[MaterializeRelation]:
+        # Adapted from: https://github.com/dbt-labs/dbt-snowflake/blob/main/dbt/adapters/snowflake/impl.py#L110
```

Comment:
```suggestion
```

Reply:
This is pretty much the core implementation: https://github.com/dbt-labs/dbt-core/blob/c270a77552ae9fc66fdfab359d65a8db1307c3f3/core/dbt/adapters/sql/impl.py#L178

Reply:
Oh, gosh, thanks for pointing this out.